corehq/apps/app_manager/static/app_manager/js/module-view.js

```diff
@@ -94,7 +94,10 @@ $(function () {
 
     // Module filter
     $(function () {
-        $('#module-filter').koApplyBindings({xpath: initial_page_data("module_filter") || ''});
+        var $moduleFilter = $('#module-filter');
+        if ($moduleFilter.length) {
```

Comment:
have to use `.length`, because `$(...)` always returns the jquery object. since the jquery object acts something like an array, i suppose it's at least consistent with the fact that empty arrays are truthy in javascript...